src/main/java/org/opentripplanner/framework/geometry/GeometryUtils.java

```diff
@@ -44,6 +48,21 @@ public class GeometryUtils {
     }
   }
 
+  public static <T> Geometry makeConvexHull(
```

Comment:
This should have JavaDoc as well, but it is moved from another place and did not have JavaDoc in the existing code - so I will not fix it in the PR.